presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeGeneralQueries.java

```diff
@@ -77,4 +77,10 @@ public void testSubqueries() {}
     @Override
     @Ignore
     public void testTopN() {}
+
+    // Ignore this case temporarily until
```

Comment:
This comment is wrong. We won't be merging https://github.com/prestodb/presto/pull/19669 as it is the same as this change. Instead, say

```
// This test is flaky: https://github.com/prestodb/presto/issues/19665
```